lib/tasks/default.rake

```diff
@@ -15,6 +15,7 @@ task(:default).clear.enhance %w(
   yaml_syntax_check
   html_from_markdown
   fasterer
+  fastly:test
```

Comment:
I would encourage you to add `fastly:test` to the `ci` task below but not to the default rake test. CircleCI [runs](https://github.com/linuxfoundation/cii-best-practices-badge/blob/master/circle.yml#L11) the `ci` task, but default rake is used on dev machines, where there's no need to require network access.

Alternatively, you might decide not to include it as an automatically run test at all.